docs/tutorial.rst

```diff
@@ -1406,10 +1413,8 @@ may have one of the following signatures:
         for i in range(5):
             time.sleep(i)
             print(i, file=stdout)
+            stdout.flush()
 
-        # In this form, the return value should be a single integer
-        # representing the "return code" of the alias (zero if successful,
-        # non-zero otherwise)
```

Comment:
I tested this function with a stdout return value and it seemed to work fine. I believe that the number of arguments is irrelevant to the allowed return values. This comment implied that the 4-arg version required an integer return, so I removed the comment.